pandas/tests/series/test_api.py

```diff
@@ -216,30 +213,6 @@ def test_empty_method(self):
         for full_series in [Series([1]), s2]:
             assert not full_series.empty
 
-    @async_mark()
-    @td.check_file_leaks
-    async def test_tab_complete_warning(self, ip):
-        # https://github.com/pandas-dev/pandas/issues/16409
-        pytest.importorskip("IPython", minversion="6.0.0")
-        from IPython.core.completer import provisionalcompleter
-
-        code = "import pandas as pd; s = Series(dtype=object)"
```

Comment:
shouldn't this be `s = pd.Series` instead of `s = Series`?  I made this change above, but a bit concerned that it isn't failing in master.

Reply:
not sure i understand the comment; the test seems to be passing?

Reply:
how is Series in the namespace?  it it getting it from the file namespace?  if so, why bother with importing pd?

I guess my concern is that it is passing because it isnt actually testing what its supposed to

Reply:
ahh i see what you mean. no guarantee that this is not currently skipped; i see you re-wrote it anyhow.